topn--2.0.0--2.1.0.sql

```diff
@@ -0,0 +1,6 @@
+DROP FUNCTION topn(jsonb, integer);
+CREATE TYPE topn AS (item text, frequency bigint);
+CREATE FUNCTION topn(jsonb, integer)
```

Comment:
We have both a function and a type with the name "topn", should we change either the name of the function or the type ?

Reply:
I think we shouldn't change the function name. We discussed so much about the APIs. We may change the type name, tho. @marcocitus any suggestions?

Reply:
Maybe topn_record or topn_counter